### ISSUE TITLE
Keyboard not switching desktops

### ISSUE THREAD

**Issue by Omgmydogisdead** [hrvach/deskhop#141]
My deskhop is working for the mouse but the keyboard is not switching desktops when the mouse moves to another desktop. This time, it's definitely not a hardware problem as I have re-checked it all. When the keyboard is plugged into board A(the default), the keyboard will only work on desktop A and never desktop B but when the keyboard is plugged into Board B, the keyboard will only work on desktop B and never desktop A. The mouse works normally in both cases.

**Comment by Omgmydogisdead** [comment]
yes, I have tried control +  CAPSLOCK. it stopped typing on Desktop A but still didn't start typing on Desktop B

**Comment by hrvach** [comment]
This is super weird and I must admit never seeing anything like this. Could you please try with another keyboard first, to rule out the keyboard being a factor here? Please describe your setup if possible. If keyboard works on either computer, that means it's being understood and parsed correctly, and if the mouse switches outputs, it means serial connection should be working. 

![image](https://github.com/user-attachments/assets/9b914434-d727-49e7-bb2a-260497394ff9)

**Comment by gothkar** [comment]
I'm experiencing this same issue. Just built two copies of the 1.1 PCB and freshly flashed. Is there some diagnostic or something I can run?

EDIT: Resolved with a reflow of U3,U4,U5
